### PR TITLE
Add DSL to setup subscription to Spree event

### PR DESCRIPTION
**Pull request by sikachu**
Usage:

    Proclaimer.configure do |config|
      # Subscribe to a particular event
      config.subscribe("order.complete") do |event, payload|
        # ...
      end

      # Subscribe to any events related to order
      config.subscribe("order") do |event, payload|
        # ...
      end

      # Subscribe to any Spree events
      config.subscribe_all do |event, payload|
        # ...
      end
    end

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,30 @@ bundle
...
+    # ...
+  end
+
+  # Subscribe to any events related to order
+  config.subscribe("order") do |event, payload|
+    # ...
+  end
+
+  # Subscribe to any Spree events
+  config.subscribe_all do |event, payload|
```
Comment (elle) [inline README.md]: What are your thoughts on instead of `subscribe_all` use `subscribe("all")`?
For consistency reasons...

**Comment by elle** [comment]
Looks ok to me, but @vinniefranco should probably give the thumbs up since he will be using this DSL

File: README.md (hunk 1 above)
Comment (sikachu) [inline README.md]: I actually copied this style from StripeEvent, and I feel like there might be time that `spree.all.[something]` is a valid event name. So, I'd rather keep it like this.

Comment (sikachu) [inline README.md]: Forgot the link. See here: https://github.com/integrallis/stripe_event#usage

**Comment by vinniefranco** [comment]
:+1:

**Pull request merged**